radix-engine/tests/blueprints/access_rules/src/lib.rs

```diff
@@ -20,26 +20,15 @@ blueprint! {
         }
 
         pub fn access_rules_method(&self) -> Vec<ComponentAccessRules> {
-            let component = Component(Runtime::actor().as_component().0);
-            component.access_rules_chain()
+            todo!("Support for self");
```

Comment:
This is exactly one of reasons for me giving up my last attempt to remove actor. :)